pkg/config/config_test.go

```diff
@@ -26,6 +26,7 @@ func TestLoadConfigSuccess(t *testing.T) {
 	t.Setenv("BOTKUBE_SETTINGS_CLUSTER__NAME", "cluster-name-from-env")
 	t.Setenv("BOTKUBE_SETTINGS_KUBECONFIG", "kubeconfig-from-env")
 	t.Setenv("BOTKUBE_SETTINGS_METRICS__PORT", "1313")
+	t.Setenv("BOTKUBE_SETTINGS_HEALTH__PORT", "1414")
```

Comment:
Once we add health endpoint under same router, we won't need dedicated port for that

Reply:
Any suggestion how to do the change?

1. Keep `BOTKUBE_SETTINGS_METRICS__PORT` and document it also covers health port configuration
2. Deprecate `BOTKUBE_SETTINGS_METRICS__PORT` and introduce more generic envar, eg `BOTKUBE_SETTINGS_OBSERV__PORT`

Reply:
🤔 That would be a breaking change, not sure if this is needed. Honestly I would stick to a separate server for now 👍 Yes, later we can have a single port for all the endpoints (GraphQL, metrics, health ones).